setup.py

```diff
@@ -47,6 +47,10 @@
     entry_points={'console_scripts': [
         'django-admin = django.core.management:execute_from_command_line',
     ]},
+    install_requires=[
+        'asgiref>=0.10',
+        'daphne>=0.11',
```

Comment:
Both of these packages have little to no test coverage. It concerns me that they are going to be core dependencies to Django.